src/Config/ObjectModel/RuntimeEntities.cs

```diff
@@ -121,9 +121,21 @@ private static Entity ProcessRestDefaults(Entity nameCorrectedEntity)
         // If no Rest node was provided in the config, set it with the default state of enabled for all verbs
         if (nameCorrectedEntity.Rest is null)
         {
-            nameCorrectedEntity = nameCorrectedEntity
-                with
-            { Rest = new EntityRestOptions(EntityRestOptions.DEFAULT_SUPPORTED_VERBS) };
+            nameCorrectedEntity = (nameCorrectedEntity.Source.Type is EntitySourceType.StoredProcedure) ?
+                                    nameCorrectedEntity
+                        with
+                                    { Rest = new EntityRestOptions(EntityRestOptions.DEFAULT_SUPPORTED_VERBS_FOR_SP) }
+                    :
+                    nameCorrectedEntity
+                        with
+                    { Rest = new EntityRestOptions(EntityRestOptions.DEFAULT_SUPPORTED_VERBS) };
+        }
+        else if (nameCorrectedEntity.Source.Type is EntitySourceType.StoredProcedure && nameCorrectedEntity.Rest.Methods.Length == 0)
+        {
+            // REST Method field is relevant only for stored procedures. For entities backed by tables and views, all the methods are supported.
```

Comment:
It would probably be more accurate to say that tables and views support all HTTP verbs by design (unless permissions config determines otherwise). For stored procedures, we still support all HTTP verbs but POST is assigned by default unless otherwise specified.

Reply:
essentially the use of the word 'support' in the comment can be updated to a word that better reflects the intended design.

Reply:
updated the comment